cmd/cmd.go

```diff
@@ -124,6 +124,7 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	bars := make(map[string]*progress.Bar)
+	var convertSpin *progress.Spinner
```

Comment:
can we re-use `spinner` here?

Reply:
I don't think we can with the current spinner. We have no way of checking the current spinner's value so we don't know whether to stop the current spinner and restart it or just update the message. We can definitely change `spinner.go` to allow that but not sure if that's what we want. Having separate spinners for each progress makes the code a bit easier to read I feel as well.